front/src/reducers/osrdconf/types.ts

```diff
@@ -29,6 +29,8 @@ export interface OsrdConfState extends InfraState {
   searchDatetimeWindow?: { begin: Date; end: Date };
   rollingStockID?: number;
   speedLimitByTag?: string;
+  totalMass?: number;
+  totalLength?: number;
```

Comment:
These fields are specific to `stdcm`, so they don't need to be in `OsrdConfState`.
```suggestion
```